src/main/controlMod.F90

```diff
@@ -629,7 +629,7 @@ subroutine control_init(dtime)
          snicar_solarspec /= 'mid_latitude_winter' .or.  &
          snicar_dust_optics /= 'sahara' .or.  &
          snicar_numrad_snw /= 5 .or.  &
-         snicar_snobc_intmix .or. snicar_snodst_intmix .or.  &
+         snicar_snodst_intmix .or.  &
```

Comment:
Good catch, thanks.

Reply:
Done

Reply:
I've rework the logic in CLMBuiildNamelist.pm because my original change wasn't working correctly.
Individually, now snicar_snobc_intmix = .false.  = .true. or .false. and snicar_snodst_intmix = .false.   We check for snicar_snodst_intmix = .true. and flag it as UNSUPPORTED in the first section of code.

The possible combinations of snicar_snobc_intmix and snicar_snodst_intmix are:

(1) snicar_snobc_intmix = .false.    snicar_snodst_intmix = .false.
(2) snicar_snobc_intmix = .true.    snicar_snodst_intmix = .false.
(3) snicar_snobc_intmix = .false.    snicar_snodst_intmix = .true.
(4) snicar_snobc_intmix = .true.    snicar_snodst_intmix = .true.

In the second section of code, (4) is flagged as "do not work together".  (1), (2), and (3) are not flagged, however, (3) is flagged as UNSUPPORTED in the first section of code because snicar_snodst_intmix is not allowed to be .true.

Tested using the various combinations in clm_short tests.
I was trying to preserve the original code as much as possible, but there may be a better way to check the valid combinations, e.g., seems like one could simply check for both .true. and flag it.

Reply:
Oh @olyson just remove snicar_snobc_intmix from the list. There's separate checking to make sure it's a valid FORTRAN logical.

So you just remove that one line, and you are good.

Reply:
I had removed that line but the code after that was flagging the valid .false., .false. case as invalid.